freesound/local_settings.example.py

```diff
@@ -50,6 +50,10 @@
 STRIPE_KEY = "sk_test_..."
 PAYPAL_EMAIL = "paypal@freesound.org"
 PAYPAL_VALIDATION_URL = "https://www.sandbox.paypal.com/cgi-bin/webscr"
+PAYPAL_PAYMENTS_API_URL = "https://api-3t.sandbox.paypal.com/nvp"
+PAYPAL_PASSWORD = ""
```

Comment:
Do we really have to use our paypal username and password? or isn't there any other token that we can generate to access the API?

Reply:
This are not the regular password and username, this are values given by paypal

Reply:
ok, where do we get these values?

Reply:
You need to go to the administration page in:

https://developer.paypal.com/developer/accounts/

then, in the left menu click on 'Accounts', after that click on the user and when a menu is displayed click on 'Profile' . Finally in the tab API Credentials you will find this info under 'Classic Sandbox API Credentials'.